### PR TITLE
[INTG-1620] Updating readme.md with FAQ

### PR DESCRIPTION
**Pull request by hkf57**
Adding a FAQ page - this is also mirrored to our support page [here](https://help.safetyculture.com/en_us/1064273113-Hyv7g_tVv).

### CODE REVIEW

**Review by phanikumarpatchigolla — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -8,6 +8,21 @@ To learn how to install and use this connector please visit our [support page](h
...
+
+## What is the purpose of the "Org Name"?
+This is the name to distinguish between your organisations locally, should you choose to consume data from multiple organisations inside iAuditor. This information is never transmitted to our servers and is only stored locally on your machine. This name can be anything you want, as long as it is unique.
+
+## Can the data be filtered when loading data for the first time?
+Yes, `inspections` and `inspection_items` can be filtered using the provided function `GetInspections` and `GetInspectionItems` respectively. These can be found in the [nav table](
+https://github.com/SafetyCulture/iAuditor-Power-Query-Connector/blob/master/iAuditor.pq#L274-L286)
+
+## What are some best practices to follow when loading data using this connector?
+We have a sample report provided which includes a few common use cases of the data present. The report can be found [here](https://github.com/SafetyCulture/iAuditor-Power-Query-Connector/blob/master/Sample%20Report.pbit)
```
Comment (BCook98) [inline README.md:21]: Can you use a relative link here instead and let GitHub complete it `./Sample%20Report.pbit`

File: README.md [hunk 2]
```diff
@@ -8,6 +8,21 @@ To learn how to install and use this connector please visit our [support page](h
...
 A Power BI report template has been included with [each release](https://github.com/SafetyCulture/iAuditor-Power-Query-Connector/releases). After installing the connector you can download and open this report in Power BI desktop. You will be prompted to enter an API key which you can generate by visiting [API Tokens](https://app.safetyculture.com/account/api-tokens) page in SafetyCulture website.
 
+# Frequently Asked Questions (FAQ)
+
+## What is the purpose of the "Org Name"?
+This is the name to distinguish between your organisations locally, should you choose to consume data from multiple organisations inside iAuditor. This information is never transmitted to our servers and is only stored locally on your machine. This name can be anything you want, as long as it is unique.
+
+## Can the data be filtered when loading data for the first time?
+Yes, `inspections` and `inspection_items` can be filtered using the provided function `GetInspections` and `GetInspectionItems` respectively. These can be found in the [nav table](
+https://github.com/SafetyCulture/iAuditor-Power-Query-Connector/blob/master/iAuditor.pq#L274-L286)
```
Comment (BCook98) [inline README.md:18]: Can you use a relative link here instead and let GitHub complete it `./Sample%20Report.pbit`

File: README.md [hunk 3]
```diff
@@ -8,6 +8,21 @@ To learn how to install and use this connector please visit our [support page](h
...
+
+## What are some best practices to follow when loading data using this connector?
+We have a sample report provided which includes a few common use cases of the data present. The report can be found [here](https://github.com/SafetyCulture/iAuditor-Power-Query-Connector/blob/master/Sample%20Report.pbit)
+
+## We are seeing many tables with no records. i.e. `schedule_assigneeers`, `groups` and `group_user`. Is this expected?
```
Comment (BCook98) [inline README.md:23]:
```suggestion
## Some tables have no data in them i.e. `schedules`, `schedule_assignees`, `groups` and `group_users`. Is this expected?
```

File: README.md (hunk 1 above)
Comment (hkf57) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (hkf57) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -8,6 +8,24 @@ To learn how to install and use this connector please visit our [support page](h
...
 A Power BI report template has been included with [each release](https://github.com/SafetyCulture/iAuditor-Power-Query-Connector/releases). After installing the connector you can download and open this report in Power BI desktop. You will be prompted to enter an API key which you can generate by visiting [API Tokens](https://app.safetyculture.com/account/api-tokens) page in SafetyCulture website.
 
+# Frequently Asked Questions (FAQ)
+## What’s the purpose of the “Org Name”?
+The “Org Name” helps distinguish between organizations locally. This can be particularly useful if you choose to consume data from multiple iAuditor organizations. The “Org Name” information doesn’t get transmitted to our servers and is only stored locally on your machine.
+
+You can label the “Org Name” freely, as long as it's unique.
+
+## Can the data be filtered when loading data for the first time?
+Yes, the “inspections” and “inspection_items” data sets can be filtered using the provided functions of “GetInspections” and “GetInspectionItems”, respectively. Both can be found in the [nav table.](./iAuditor.pq#L274-L286)
```
Comment (atashbahar) [inline README.md:18]: I'm a bit worried about including line number (`./iAuditor.pq#L274-L286`) here. Can get out of date quickly.

**Review by atashbahar — COMMENTED** [review]
This being a public repo, it would be great to have a description for the PR.

File: README.md (hunk 4 above)
Comment (hkf57) [reply README.md:18]: Any proposals for this..? Does github support anchoring of any sort?

Comment (atashbahar) [reply README.md:18]: I'm not sure if it does. But I don't see any point referring to the code. You can get rid of the last sentence.

Comment (hkf57) [reply README.md:18]: I think its OK for now, at least it links somewhere.

Comment (atashbahar) [reply README.md:18]: My understanding is this FAQ is more targeted towards the end consumer. They just use the connector as is and never bother with the code.
When we say `GetInspections` and `GetInspectionItems` functions. It's clear that we are referring to the functions that they see when connect to iAuditor in Power BI.
Having those code line numbers will only confuse these customers.

Comment (hkf57) [reply README.md]: That's a very good point, I was thinking of aiming this section both for developers as well as plug and play users that are a little curious. Removed nonetheless.

**Review by atashbahar — APPROVED** [review]

**Review by BCook98 — APPROVED** [review]

**Pull request merged**